CHANGES.rst

```diff
@@ -52,6 +52,9 @@ astropy.stats
 astropy.table
 ^^^^^^^^^^^^^
 
+- Added functionality to allow ``rename_columns`` which takes in multiple
+  column names and updates the table with the new column names. [#5159]
```

Comment:
Please also add a mention of `#8070` in the `[]` to make the bot happy. Thanks! 😄

Reply:
Sure, I've added this in the latest commit